crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -97,8 +145,12 @@ async fn protocol_fee_test_case(test_case: TestCase) {
     let test: Test = tests::setup()
         .name(test_name)
         .pool(pool)
+        .jit_order(jit_order.clone())
         .order(order.clone())
```

Comment:
Do we need both `order` and `jit_order` if the latter is `Some`? Maybe we could add a flag to the Order itself whether it's JIT or not.

Reply:
They differ because JIT orders are going to be sent just from a particular solver (not all), so it has to be set differently.